debugProtocol.json

```diff
@@ -1273,7 +1273,7 @@
 					"description": "The value of the variable."
```

Comment:
Should we update the description of "name" to indicate that it is valid to pass the "evaluateName"?